packages/components/src/styles/display/display.stories.ts

```diff
@@ -1,22 +1,21 @@
 import '../../solid-components';
-
+import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
 
 const { argTypes, parameters } = storybookDefaults('sd-display');
 const { overrideArgs } = storybookHelpers('sd-display');
 const { generateTemplate } = storybookTemplate('sd-display');
 
 /**
- * Display provides larger text sizes that are not used as headlines. <br>
- * <br>
- * The different sizes allow for a more versatile styling of text elements. Display text should not be used as substitute for headlines.<br>
- * <br>
- * <b>Sizes</b>
- * <li>xl is the default size.</li>
+ * Used to provide larger text sizes that are not used as headlines.
+ *
+ * **Related templates**:
+ * - [Headline, Display and Leadtext with Mark](?path=/docs/templates-headline-display-and-leadtext-with-mark--docs)
```

Comment:
Template name should be: "Headline with Mark"

Reply:
Updated